spec/ddtrace/contrib/mysql2/patcher_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+
+require 'ddtrace'
+require 'mysql2'
+
+RSpec.describe 'Mysql2::Client patcher' do
+  let(:tracer) { Datadog::Tracer.new(writer: FauxWriter.new) }
+
+  let(:client) do
+    Mysql2::Client.new(
+      host: host,
+      port: port,
+      database: database,
+      password: 'root'
```

Comment:
I think this might need to be `let(:password) { ENV.fetch('TEST_MYSQL_ROOT_PASSWORD', 'root') }` so as to avoid configuration issues in our test environment.

Reply:
I mimicked the active record spec and added an explicit username and the password var.